Packs/StellarCyber/Integrations/StellarCyber/StellarCyber.yml

```diff
@@ -0,0 +1,275 @@
+category: Analytics & SIEM
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: StellarCyber
+  version: -1
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+- display: Incident type
+  name: incidentType
+  type: 13
+- display: Mirroring Direction
+  name: mirror_direction
+  type: 15
+  section: Collect
+  required: false
+  additionalinfo: 'If set to Incoming, will mirror Cases from Stellar Cyber to XSOAR. If set to None, will not mirror Cases from Stellar Cyber to XSOAR. Default is None.'
+  defaultvalue: None
+  options:
+  - None
+  - Incoming
+- name: stellar_dp
+  display: Stellar Cyber Host (e.g. example.stellarcyber.cloud)
+  required: true
+  defaultvalue: example.stellarcyber.cloud
+  type: 0
+  section: Connect
+  additionalinfo: "Your Stellar Cyber Host FQDN."
+- name: credentials
+  display: API User (Email Address)
+  required: true
+  type: 9
+  displaypassword: API Key
+  hiddenusername: false
+  section: Connect
+- name: first_fetch
+  display: First fetch time
+  required: false
+  defaultvalue: 1 day
+  type: 0
+  section: Collect
+  additionalinfo: "The time to look back for initial pull of cases. Example values: 1 day, 5 hours, 30 minutes, etc."
```

Comment:
Time/date formats must add examples to the descriptions of the fields (i.e. ISO8601 as 2020-01-01T00:11:22Z)